crypto/keys/secp256k1/secp256k1.go

```diff
@@ -21,16 +21,74 @@ import (
 
 var (
 	_ cryptotypes.PrivKey  = &PrivKey{}
+	_ cryptotypes.PrivKey  = &PrivKeyOld{}
 	_ codec.AminoMarshaler = &PrivKey{}
+	_ codec.AminoMarshaler = &PrivKeyOld{}
 )
 
 const (
-	PrivKeySize = 32
-	keyType     = "secp256k1"
-	PrivKeyName = "tendermint/PrivKeySecp256k1"
-	PubKeyName  = "tendermint/PubKeySecp256k1"
+	PrivKeySize    = 32
+	keyType        = "secp256k1"
+	PrivKeyNameOld = "tendermint/PrivKeySecp256k1"
+	PubKeyNameOld  = "tendermint/PubKeySecp256k1"
+	PrivKeyName    = "comet/PrivKeySecp256k1Uncompressed"
+	PubKeyName     = "comet/PubKeySecp256k1Uncompressed"
 )
 
+func (privKey *PrivKeyOld) Bytes() []byte {
```

Comment:
I performed an ad-hoc diff-of-diffs of this directory comparing it to the analogous code from 0xPolygon/cometbft#2.
I think it would ease maintenance if these functions resembled the ones[here](https://github.com/0xPolygon/cometbft/blob/62a78ac1/crypto/secp256k1/secp256k1.go#L44-L63).

Reply:
Yeah that was my direction as well, but I tried to keep the diffs with upstream as small as possible.
For example, I did not change the order of the methods in `crypto/keys/secp256k1/secp256k1.go`.
Also, some functions are left to `crypto/keys/secp256k1/secp256k1_nocgo.go` where the original methods of the type (privKey *PrivKey) were defined (e.g. `Sign` and `VerifySignature`)